modules/iam-group-with-policies/policies.tf

```
@@ -35,7 +35,10 @@ data "aws_iam_policy_document" "iam_self_management" {
       "iam:GetUser"
     ]
 
-    resources = ["arn:${local.partition}:iam::${local.aws_account_id}:user/$${aws:username}"]
+    resources = [
+      "arn:${local.partition}:iam::${local.aws_account_id}:user/$${aws:username}",
```

we should add a variable `path` that defaults to `"/"` and then update like:
```suggestion
["arn:${local.partition}:iam::${local.aws_account_id}:user/${var.path}$${aws:username}"]
```

---

Adding a path here will not work as users with different paths (or even with no path) can be added to the same IAM group.

---

ah right, you are correct. It would have to be something like `$${aws:userpath}` that AWS would know how to populate dynamically